README.md

```diff
@@ -23,12 +23,15 @@ hEl.addElement({
   y: 500,
   width: 4000,
   height: 2500,
-  (optional)fontSize: 14
+  (optional)fontSize: 14,
+  (optional)animation: "zoom"|"pan"
```

Comment:
I would rather this not to be a property, as the element should be agnostic to any changes the world makes, even if related to him. This could be passed as a property to the `goToElementLocation` method.

Reply:
For me it doesn't really matter which way it's done so I'll go ahead and make the change so that it's a parameter on the goToElementLocation method. 

I was thinking that there could be the potential to want to "animate" to different overlays differently and the way I am adding overlays dynamically, I thought it made sense to add it as a property to the overlay element.

Reply:
Sure, but that still is only a way to modify the behaviour of the goTo method, the actual element doesn't rely on this value for anything.